test/unit/mdc-menu-surface/mdc-menu-surface.test.js

```diff
@@ -26,7 +26,7 @@ import bel from 'bel';
 import domEvents from 'dom-events';
 import td from 'testdouble';
 
-import {MDCMenuSurface, MDCMenuSurfaceFoundation} from '../../../packages/mdc-menu-surface/index';
+import {MDCMenuSurface, MDCMenuSurfaceFoundation} from '../../../packages/mdc-menu-surface';
```

Comment:
revert change

Reply:
The tests pass as-is, so why does it matter?

Reply:
Without `/index`, any node toolchain will resolve this to our UMD code under `dist` (due to `main` in package.json) which is probably not what they want.

Reply:
Yeah but this is a unit test, not a source file, and we're pointing to `packages/mdc-menu-surface`, not `@material/menu-surface` (so there is no `dist`). Is it really still a problem?

I see your point for source files, though 🙂